news/2018-09-18-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: September 2018"
+date: 2018-10-18 12:00:00 +0500
+---
+
+Huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and a brand new contributor? What will they come up with next? This month’s issue of the QAT Gazette has all the answers!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+A new month brings new adventure, new challenges, and a new face to our humble blog!
+
+A lot has happened in the past month and as you might expect, you can find it all right here! From substantial changes to the way Beatmap Nominator applications are conducted, to ongoing developments within the Modding Discussions system and of course, some aforementioned new faces, keep reading to find out everything you need to know in and around the mapping and modding community.
+
+## Within the Beatmap Nominators
+by [Kibbleru](https://osu.ppy.sh/users/3193504)
+
+We had yet another exciting month within the Beatmap Nomination Group. The long-awaited new round of Beatmap Nominator applications were finally released in late September, but this time, we have made some significant changes to both the application and evaluation processes. Continue reading to find out more!
```

Comment:
were finally -> was finally
i think

Reply:
Hmm I suppose so yeah, I'm used to a "round" being used to describe multiple things (which it could technically still be doing for the 4 game modes), but we're unifying things so singular probably makes more sense.